src/myrtlespeech/model/deep_speech_2.py

```diff
@@ -134,10 +137,15 @@ def forward(
         Args:
             x: Input for the cnn module. See initialisation docstring.
 
+            hx: An Optional RNNState of ``self.rnn`` (see :py:class:`RNN` for
+                details.)
+
         Returns:
-            Output from the fully connected layer but with the ``batch`` and
+            A ``Tuple[fc_output, state]``. ``fc_output`` is the result
+            after applying the fully connected layer but with the ``batch`` and
```

Comment:
This docstring is for FC and not DeepSpeech2?

Reply:
This is referring to the DeepSpeech2 fully_connected layer. I've updated the docstring to make this clearer.